backend/fms_core/serializers.py

```diff
@@ -672,21 +678,34 @@ class Meta:
             'pedigree',
             ]
 
+class StepSpecificationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = StepSpecification
+        fields = ("id", "display_name", "sheet_name", "column_name", "value")
+
 class StepSerializer(serializers.ModelSerializer):
     class Meta:
         model = Step
         fields = ["id", "name", "protocol_id"]
 
-class WorkflowSerializer(serializers.ModelSerializer):
-    steps = serializers.SerializerMethodField()
+class StepOrderSerializer(serializers.ModelSerializer):
+    protocol_id = serializers.CharField(read_only=True, source='step.protocol_id')
+    step_id = serializers.CharField(read_only=True, source='step.id')
+    step_name = serializers.CharField(read_only=True, source='step.name')
+    class Meta:
+        model = StepOrder
+        fields = ["id", "protocol_id", "step_id", "step_name", "order"]
 
+class WorkflowSerializer(serializers.ModelSerializer):
+    steps = serializers.SerializerMethodField(read_only=True)
     class Meta:
         model = Workflow
         fields = ("id", "name", "structure", "steps")
-
+    
     def get_steps(self, instance):
-        steps = instance.steps.all().order_by("StepsOrder__order")
-        return StepSerializer(steps, many=True).data
+        steps = instance.steps.all().order_by("steps_order__order")
+        serialized_data = StepSerializer(steps, many=True)
```

Comment:
Workflow should use the step_order serializer if possible ...

Reply:
Apparently Chris doesn't need the step order for the Workflow endpoint.

Reply:
It would be cleaner for the workflow to reference the step_order than the step (the order of the step being directly related to the step_order anyway). By definition the workflow is a sequence (or an ordering) of steps. If we ever have to add information to the workflow it's likely to be in the workflow or step_order models. The step model is shared between workflows and if we include it we may come to have fields that conflict.